doc/references.bib

```diff
@@ -0,0 +1,22 @@
+% Encoding: UTF-8
+@article{AvantsEtAl2008,
+  author = {Avants, Brian B. and Epstein, Charles L. and Grossman, Murray C. and Gee, James C.},
+  doi = {10.1016/j.media.2007.06.004},
+  journal = {Medical Image Analysis},
+  number = {1},
+  pages = {26-41},
+  shorttitle = {Symmetric Diffeomorphic Image Registration with Cross-Correlation},
+  title = {Symmetric Diffeomorphic Image Registration with Cross-Correlation: Evaluating Automated Labeling of Elderly and Neurodegenerative Brain},
+  volume = {12},
+  year = {2008}
+}
+
+@book{OppenheimEtAl1999,
+  address = {Upper Saddle River, NJ},
+  edition = {2 edition},
+  title = {Discrete-{Time} {Signal} {Processing}},
+  isbn = {978-0-13-754920-7},
+  publisher = {Prentice Hall},
+  author = {Oppenheim, Alan V. and Schafer, Ronald W. and Buck, John R.},
+  year = {1999}
+}
```

Comment:
To be removed?

Reply:
Oh yeah good catch, I need to replace this with the ICLabel publication and add to the docstring in #21 for `label_ic` and the `ICLabelNet`.